diff -Nur docbook-dsssl-1.54/cygnus/cygnus-both.dsl docbook-dsssl-1.54.new/cygnus/cygnus-both.dsl
--- docbook-dsssl-1.54/cygnus/cygnus-both.dsl	Thu Mar 30 15:35:40 2000
+++ docbook-dsssl-1.54.new/cygnus/cygnus-both.dsl	Wed May 10 20:39:38 2000
@@ -2,11 +2,11 @@
 <!ENTITY % html "IGNORE">
 <![%html;[
 <!ENTITY % print "IGNORE">
-<!ENTITY docbook.dsl SYSTEM "/usr/share/sgml/dsssl/html/docbook.dsl" CDATA dsssl>
+<!ENTITY docbook.dsl PUBLIC "-//Norman Walsh//DOCUMENT DocBook HTML Stylesheet//EN" CDATA dsssl>
 ]]>
 <!ENTITY % print "INCLUDE">
 <![%print;[
-<!ENTITY docbook.dsl SYSTEM "/usr/share/sgml/dsssl/print/docbook.dsl" CDATA dsssl>
+<!ENTITY docbook.dsl PUBLIC "-//Norman Walsh//DOCUMENT DocBook Print Stylesheet//EN" CDATA dsssl>
 ]]>
 ]>
 
