--- docbook-dsssl-1.76/common/dbcommon.dsl.orig	Fri Feb 22 11:26:14 2002
+++ docbook-dsssl-1.76/common/dbcommon.dsl	Fri Feb 22 11:43:00 2002
@@ -1070,11 +1070,16 @@
 (define (article-title nd)
   (let* ((artchild  (children nd))
 	 (artheader (select-elements artchild (normalize "artheader")))
+	 (articleinfo (select-elements artchild (normalize "articleinfo")))
 	 (ahtitles  (select-elements (children artheader) 
 				     (normalize "title")))
+	 (aititles  (select-elements (children articleinfo) 
+				     (normalize "title")))
 	 (artitles  (select-elements artchild (normalize "title")))
 	 (titles    (if (node-list-empty? artitles)
-			ahtitles
+			(if (node-list-empty? ahtitles)
+			    aititles
+			    ahtitles)
 			artitles)))
     (if (node-list-empty? titles)
 	""
